pkg/shell/hosts.jsx

```diff
@@ -184,13 +184,13 @@ export class CockpitHosts extends React.Component {
                                     <HostLine user={user} host={label} />
                                 </span>
                             </span>
-                            {this.state.opened && <CaretUpIcon size="md" className="pf-c-select__toggle-arrow mobile_v" aria-hidden="true" />}
-                            {!this.state.opened && <CaretDownIcon size="md" className="pf-c-select__toggle-arrow mobile_v" aria-hidden="true" />}
+                            {this.state.opened && <CaretUpIcon className="pf-c-select__toggle-arrow mobile_v pf-c-icon pf-m-lg" aria-hidden="true" />}
+                            {!this.state.opened && <CaretDownIcon className="pf-c-select__toggle-arrow mobile_v pf-c-icon pf-m-lg" aria-hidden="true" />}
```

Comment:
Is this `pf-c-icon` intentional? Seems weird that a `<FooIcon>` component doesn't do this by default?

Reply:
It's intentional. They removed the 'size' property from these *Icons, and this seems the only way I can get it to use that old sizing. 

I reported an issue and asked there for suggestion. https://github.com/patternfly/patternfly-react/issues/8990